Ginger/Ginger/Agents/AgentEditPage.xaml.cs

```diff
@@ -118,7 +118,7 @@ private void SetDriverInformation()
                 xVirtualAgentsPanel.Visibility = Visibility.Visible;
                 xAgentVirtualSupported.Content = "Yes";
 
-                VirtualAgentCount.Content = mAgent.VirtualAgentsStarted().Count;
+                //VirtualAgentCount.Content = mAgent.VirtualAgentsStarted().Count;
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=6410968)